{
	"id": "CVE-2023-2088",
	"description": "A flaw was found in OpenStack due to an inconsistency between Cinder and Nova. This issue can be triggered intentionally or by accident. A remote, authenticated attacker could exploit this vulnerability by detaching one of their volumes from Cinder. The highest impact is to confidentiality.",
	"debian_bug": 1035932,
	"scope": "local",
	"packages": [
		{
			"name": "python-os-brick",
			"status": "resolved",
			"urgency": "not yet assigned",
			"fixed_version": "4.1.0-3",
			"repository": [
				{
					"name": "trixie",
					"version": "6.7.0-2"
				}
			]
		}
	]
}
